.yarnrc.yml

```diff
@@ -0,0 +1 @@
+nodeLinker: node-modules
```

Comment:
What does this do?

Reply:
Not sure, more autogenerated code I think, can revert